src/screens/AuthedAppTileScreen.tsx

```diff
@@ -51,18 +49,15 @@ export const AuthedAppTileScreen = ({ navigation, route }: Props) => {
     }
 
     // These params will only be needed for LifeOmic app tiles
```

Comment:
We probably don't need this comment anymore.

Reply:
Yup, good call. Removed.